src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarBidAction.tsx

```diff
@@ -112,7 +112,7 @@ export class ArtworkSidebarBidAction extends React.Component<
       openAuthModal(ModalType.login, {
         redirectTo,
         intent: Intent.bid,
-        copy: "Log in to bid on artworks",
+        copy: "Sign up to bid on artworks",
```

Comment:
This looks wrong to me. The log-in form is being shown (`ModalType.login`), so the Sign up language is confusing. Maybe the intent was to show the sign-up form instead?